### PR TITLE
Add inpainting test

### PR DESCRIPTION
**Pull request by jinchen62**

### CODE REVIEW

File: build_tools/stable_diffusion_testing.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import argparse
 from glob import glob
 import shutil
+import wget
```
Comment (powderluv) [inline build_tools/stable_diffusion_testing.py:11]: requests please

Comment (powderluv) [reply build_tools/stable_diffusion_testing.py:11]: I mean please use `import requests`

File: build_tools/stable_diffusion_testing.py [hunk 2]
```diff
@@ -64,6 +65,75 @@ def test_loop(device="vulkan", beta=False, extra_flags=[]):
...
+        "https://huggingface.co/datasets/diffusers/test-arrays/resolve"
+        "/main/stable_diffusion_inpaint/input_bench_mask.png"
+    )
+    img = requests.get(img_url)
+    mask = requests.get(mask_url)
+    open("./test_images/input/image.png", "wb").write(img.content)
+    open("./test_images/input/mask.png", "wb").write(mask.content)
+
+
+def test_inpaint(device="vulkan", beta=False, extra_flags=[]):
```
Comment (dan-garvey) [inline build_tools/stable_diffusion_testing.py:84]: There is a lot of duplicated code, please add an argument that specifies inpainting and use conditionals where necessary

**Review by dan-garvey — COMMENTED** [review]
Merge the two functions

**Review by dan-garvey — APPROVED** [review]

**Pull request merged**